### PR TITLE
GET route for `publish`

### PR DESCRIPTION
**Pull request by AparnaKarve**
A simple GET route `'/api/v0/publish'` was added which would be used by the AI Service to do it's 
liveness/readiness check

### CODE REVIEW

File: tests/test_wsgi.py [hunk 1]
```diff
@@ -53,3 +54,19 @@ def test_route_with_upload_service_absent(self, mocker):
...
+    def test_get_route_for_publish(self, mocker):
+        """Test index route when upload service is present."""
+        client = application.test_client(mocker)
+
+        url = '/api/v0/publish'
+
+        response = client.get(url)
+        assert response.get_data() == b'{"message":"Requires a POST call ' \
+            b'to publish recommendations",' \
+            b'"status":"OK","version":"0.0.1"}\n'
```
Comment (jameswnl) [inline tests/test_wsgi.py]: Can you use the `VERSION`?

File: tests/test_wsgi.py [hunk 2]
```diff
@@ -49,7 +60,31 @@ def test_route_with_upload_service_absent(self, mocker):
...
+        output = {
+            "message": "upload-service not operational",
+            "status": "Error",
+            "version": VERSION
+        }
+        assert json.loads(response.get_data()) == output
         assert response.status_code == 500
+
+
+class TestGetPublish:
```
Comment (jameswnl) [inline tests/test_wsgi.py]: don't quite see we need a separate class for this test. Probably don't even need any classes in this suite.

Comment (AparnaKarve) [reply tests/test_wsgi.py]: 2 options -

We could
- either remove the class for this test, and just have a function
- or rename the class to `TestPublish` ( and add the test for POST publish separately)

How about we do the second?

Comment (jameswnl) [reply tests/test_wsgi.py]: So the second will have us only 1 class in this file, right?

Comment (AparnaKarve) [reply tests/test_wsgi.py]: yes, one class for testing the Publish route for GET and POST

Comment (AparnaKarve) [reply tests/test_wsgi.py]: nevermind, I have changed that to be a function

**Pull request merged**